lib/ansible/modules/cloud/docker/docker_config.py

```diff
@@ -80,7 +80,7 @@
 - name: Create config foo
   docker_config:
     name: foo
-    data: Hello World!
+    data: "{{ lookup('file', '/path/to/config/file') }}"  # The file is taken from the control machine
```

Comment:
I would move the comment to its own line (probably before `data:`), so that for small screen widths this won't scroll out of the viewable area.